conf/base.config

```diff
@@ -34,6 +34,14 @@ process {
     memory = { check_max( 200.GB * task.attempt, 'memory' ) }
     time = { check_max( 5.h * task.attempt, 'time' ) }
   }
+  withName: makeSalmonIndex {
```

Comment:
maybe it is worth it to create a label `salmon` with the `cpus` line and then label the process in main.nf as `salmon` and `low_memory`?